Poker.BE/Poker.BE.Domain/Game/Round.cs

```diff
@@ -212,47 +212,67 @@ private void Call(int amountToBetOrCall)
                 lastPartialPot = partialPotIterator;
                 partialPotIterator = partialPotIterator.PartialPot;
             }
-            
+
             if (amountToBetOrCall + playerCurrentBet > lastPartialPot.AmountToClaim)
-                throw new WrongIOException("Not enough partial pots were created! Somthing isn't right!");
-            
-            if (CurrentPlayer.Wallet.amountOfMoney == 0) //if call all-in move
+                throw new WrongIOException("Not enough partial pots were created! Something isn't right!");
+
+            if (CurrentPlayer.Wallet.AmountOfMoney == 0) //if call all-in move
             {
                 //Add new partial pot in the middle
-                Pot newPartialPot = new Pot(lastPartialPot.BasePot);
-                newPartialPot.PartialPot = lastPartialPot;
+                Pot newPartialPot = new Pot(lastPartialPot.BasePot)
+                {
+                    PartialPot = lastPartialPot
+                };
                 lastPartialPot.BasePot = newPartialPot;
+
                 if (newPartialPot.BasePot != null)
+                {
                     newPartialPot.BasePot.PartialPot = newPartialPot;
+                }
                 else
-                    this.currentPot = newPartialPot;
-                
+                {
+                    currentPot = newPartialPot;
+                }
+
                 //Update partial pot's fields
                 foreach (Player player in lastPartialPot.PlayersClaimPot)
+                {
                     newPartialPot.PlayersClaimPot.Add(player);
+                }
+
                 lastPartialPot.PlayersClaimPot.Remove(CurrentPlayer);
 
                 newPartialPot.AmountToClaim = amountToAdd + lastPlayerCurrentBet;
                 lastPartialPot.AmountToClaim -= newPartialPot.AmountToClaim;
                 newPartialPot.Value = lastPartialPot.Value - (lastPartialPot.AmountToClaim * lastPartialPot.PlayersClaimPot.Count);
                 lastPartialPot.Value = lastPartialPot.AmountToClaim * lastPartialPot.PlayersClaimPot.Count;
 
```

Comment:
I'm not sure that I understand the link between pot and partialpot. Do we have some pot tests?